Source/JavaScriptCore/wasm/WasmOMGIRGenerator.cpp

```diff
@@ -5308,9 +5308,7 @@ Expected<std::unique_ptr<InternalFunction>, String> parseAndCompileOMG(Compilati
     // optLevel=1.
     procedure.setNeedsUsedRegisters(false);
     
-    procedure.setOptLevel(isAnyBBQ(compilationMode)
```

Comment:
Calling this out specifically, I am thinking we don't need a separate option for this anymore since we don't do BBQB3